src/data/listen.js

```diff
@@ -20,9 +20,7 @@ const tokenWarning = [
 const printTokenWarning = once(() => console.warn(tokenWarning.join(' ')))
```

Comment:
This one should now be removed as we actually support tokens now.

Reply:
so In that case we should remove https://github.com/sanity-io/client/blob/4719587794b5a73a3356dd99a818344bb26c179d/src/data/listen.js#L15-L18

and 
https://github.com/sanity-io/client/blob/4719587794b5a73a3356dd99a818344bb26c179d/src/data/listen.js#L52-L54 

as well? Since that's where that line is used? :)

Reply:
Yes! 👍